docs/examples/exporting_models_json_nested_encoders.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime, timedelta
+from pydantic import BaseModel
+from pydantic.json import timedelta_isoformat
+
+
+class WithCustomEncoders(BaseModel):
+    dt: datetime
+    diff: timedelta
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: v.timestamp(),
+            timedelta: timedelta_isoformat,
+        }
+
+
+class ParentWithCustomEncoders(BaseModel):
+    diff: timedelta
+    child: WithCustomEncoders
+
+    class Config:
+        json_encoders = {
+            timedelta: lambda v: v.total_seconds(),
+            WithCustomEncoders: lambda _: 'using parent encoder',
+        }
```

Comment:
I think this makes the example more complicated than it needs to be - I think it's sufficient just to show that `ParentModel` uses `child.Config.json_encoders`.

If there's a more subtle point to be made about how `ParentModel` interacts, either mention it in the code or add another example.

Reply:
I think this was highlighting that `ParentModel` doesn't use `child.Config.json_encoders` when `models_as_dict=False`, whether `use_nested_encoders=True|False`. That's mentioned in the code as a comment, should it be more explicit?